src/check/constrain/generate/definition.rs

```diff
@@ -91,7 +95,11 @@ pub fn gen_def(
             let unassigned: Vec<String> = body_env
                 .unassigned
                 .iter()
-                .map(|v| format!("Non nullable class variable '{v}' not assigned to in constructor"))
+                .map(|v| if let Some(class) = &env.class {
+                    format!("Non nullable attribute '{v}' of {class} not assigned to in constructor")
```

Comment:
Should we actually not also check that we call all parent constructors?
Perhaps good to make an issue of this and add this in separate PR.